### PR TITLE
Add GitHub issue template

### PR DESCRIPTION
**Pull request by mcktr**
This adds an issue template for GitHub.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+3.
+4.
+
+## Context
+<!--- How has this issue affected you? What are you trying to accomplish? -->
+<!--- Providing context helps us come up with a solution that is most useful in the real world -->
+
+## Your Environment
+<!--- Include as many relevant details about the environment you experienced the problem in -->
+* Icinga Web 2 Grafana version (System - About):
```
Comment (nunofernandes) [inline .github/ISSUE_TEMPLATE.md]: This module doesn't require anything related to Grafana. This line should be removed.

**Review by nunofernandes — CHANGES_REQUESTED** [review]
Everything seems ok.. Just the part related to grafana should be removed.

**Comment by mcktr** [comment]
Good catch :+1:  I amended the changes.

**Comment by nunofernandes** [comment]
Merged...

Thanks!

**Pull request merged**